unit_test/batched/Test_Batched.hpp

```diff
@@ -2,6 +2,9 @@
 #define TEST_BATCHED_HPP
 
 // Serial kernels
+#include "Test_Batched_SerialAxpy.hpp"
+#include "Test_Batched_SerialAxpy_Real.hpp"
+#include "Test_Batched_SerialAxpy_Complex.hpp"
```

Comment:
Might be good to add separate test for sparse, to keep test times of each test low and run them in parallel.